.coveragerc

```diff
@@ -9,6 +9,11 @@ source =
 skip_covered = true
 skip_empty = true
 show_missing = true
+exclude_also =
```

Comment:
It looks like `covdefaults` handles all of these even better. I've been meaning to integrate it for a while...

Reply:
I'll test it with #1085 and get in, once I know it doesn't break the current setup.

Reply:
Personally, I prefer the control here. There's atleast one match in covdefaults that I don't want enabled. Seems like overkill for something that can be done with 3 simple regexes.

Reply:
Reviewing it again, I think there's multiple rules I don't want from covdefaults, and all the others we don't already use, we probably will never need.

But, this rule looks like a really bad idea to me:
`    -> ['"]?NoReturn['"]?:`
I'm pretty sure that disables coverage of entire functions just because they are marked as not returning.
Just because they don't return, doesn't mean they shouldn't be run. We have dozens of such functions across our codebases and they should not be ignored.

Reply:
@Dreamsorcerer that's an interesting corner case, I'll compare the coverage in that PR and see if it affects anything...

Reply:
FYI, I ended up merging it, will re-evaluate later if needed. Marking this thread as resolved.